functions/model/meet/lib.js

```diff
@@ -147,3 +161,20 @@ exports.updateMeetDetailsAtWorktez= function(updateJson, meetDocId) {
   const updateMeet = db.collection("Meet").doc(meetDocId).update(updateJson);
   return Promise.resolve(updateMeet);
 };
+
+
+/**
+ * Description
+ * @param {any} orgDomain
+ * @param {any} teamName
+ * @param {any} uid
+ * @param {any} docId
+ * @return {any}
+ */
+ exports.getUserMeetDetailsById = function(uid, docId) {
+  const getUserMeetDetailsById = db.collection("Users").doc(uid).collection("Meet").doc(docId).get().then((doc) => {
+    const data = doc.data();
+    return data;
+  });
+  return Promise.resolve(getUserMeetDetailsById);
+};
```

Comment:
What is the use of this API when the same data can be fetched using 
`exports.getWorktezMeetDetails` API?

Reply:
Here we are getting users meet.